HC.Tomek/HC.Tomek.Js.Ex3/containers.js

```diff
@@ -0,0 +1,61 @@
+function startPage() {
+    document.getElementById("quiz").style.display = "none";
+    document.getElementById("result").style.display = "none";
+    var startContainer = document.getElementById("start");
+    var startButton = document.createElement("button");
+    startButton.innerHTML = "Start";
+    startContainer.appendChild(startButton);
+    startButton.addEventListener("click", function () {
+        document.getElementById("start").style.display = "none";
```

Comment:
czyli jednak dodawałeś już eventHandlery :) w jquery robi się to poprzez $('button').click(function(event){    });